From 7ff956e4088e0bdc6bfd429f99124a8a9256c181 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 6 Aug 2022 21:44:11 -0700
Subject: [PATCH] libostree: Remove including sys/mount.h

This conflicts with linux/mount.h which is included by linux/fs.h
with glibc 2.36+

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/libostree/ostree-sysroot-deploy.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libostree/ostree-sysroot-deploy.c b/src/libostree/ostree-sysroot-deploy.c
index 2dc9f58b..61b19e42 100644
--- a/src/libostree/ostree-sysroot-deploy.c
+++ b/src/libostree/ostree-sysroot-deploy.c
@@ -23,7 +23,6 @@
 #include <gio/gunixoutputstream.h>
 #include <glib-unix.h>
 #include <stdint.h>
-#include <sys/mount.h>
 #include <sys/statvfs.h>
 #include <sys/socket.h>
 #include <sys/ioctl.h>
-- 
2.37.1

